en/chromebook_setup/instructions.md

```diff
@@ -61,6 +61,77 @@ This bottom area is your terminal. You can use the terminal to send instructions
 to the remote Cloud 9 computer. You can resize that window to make it a bit
 bigger.
 
+#### Glitch.com Cloud IDE
+
+1. Go to [Glitch.com](https://glitch.com/)
+2. Sign up for an account (https://glitch.com/signin) or use your Github account if you have one (see Github instructions below).
+3. Click _New Project_ and choose _Hello Webpage_
+4. Click Terminal button (on the left side of the window)
+
+When using Glitch.com as your Cloud IDE, we do not create a virtual environment but we create the following files manually :
+
+{% filename %}glitch.json{% endfilename %}
+```
+{
+  "install": "pip3 install -r requirements.txt --user",
+  "start": "bash start.sh",
+  "watch": {
+    "throttle": 1000
+  }
+}
+```
+
+{% filename %}requirements.txt{% endfilename %}
+```
+Django>=2.2.4
```

Comment:
To make it easier to keep this version up to date, please use the existing GitBook variable. Also, prefer `~=` (greatest compatible version) over `>=` (greatest version greater or equal to).
```suggestion
Django~={{ book.django_version }}
```